sql/1.0xx/activity_feed.sql

```diff
@@ -0,0 +1,13 @@
+BEGIN;
+
+CREATE TABLE "activity_feed" (
+  "id" serial NOT NULL,
+  "created" timestamp with time zone NOT NULL,
+  "type" text NOT NULL,
+  "description" text NOT NULL,
+  "format" text DEFAULT 'markdown' NOT NULL,
+  "for_user" bigint,
```

Comment:
If `for_user` is null, it's for everyone. I've started working on a `for_role` where you can notify classes of users (i.e. admins, community leaders, translation managers...). If we have to add 300 columns for a single notification I think it's broken.

I don't think we need a 'sent' column - if the mailer job runs hourly we should select stuff which happened in the last hour (rounded down to $hour:00:00).